tests/tests.js

```diff
@@ -5,6 +5,27 @@
 var test = require('tape');
 var ES = require('es-abstract/es6');
 
+var makeIterator = function (array) {
+	var nextIndex = 0;
+	var value;
+	return {
+		'next': function () {
+			if (nextIndex < array.length) {
+				value = array[nextIndex];
+				nextIndex = nextIndex + 1;
+				return {
+					'value': value,
+					'done': false
+				};
+			} else {
+				return {
+					'done': true
```

Comment:
I believe the [spec](https://tc39.github.io/ecma262/#sec-createiterresultobject) always creates a `value` property explicitly set to `undefined` - ie, `'value' in iteratorResult` must always be true.